prefs.js

```diff
@@ -17,15 +17,24 @@ var Prefs = class Prefs {
         this.KEY_HIBERNATE_WORKS_CHECK = "hibernate-works-check";
         this._schemaName = "org.gnome.shell.extensions.hibernate-status-button";
 
-        let schemaDir = Me.dir.get_child('schemas').get_path();
+        // first try developer local schema
+        try {
+            let schemaDir = Me.dir.get_child('schemas').get_path();
 
-        let schemaSource = Gio.SettingsSchemaSource.new_from_directory(
-            schemaDir, Gio.SettingsSchemaSource.get_default(), false
-        );
-        let schema = schemaSource.lookup(this._schemaName, false);
+            let schemaSource = Gio.SettingsSchemaSource.new_from_directory(
+                schemaDir, Gio.SettingsSchemaSource.get_default(), false
+            );
+            let schema = schemaSource.lookup(this._schemaName, false);
+
+            this._setting = new Gio.Settings({
+                settings_schema: schema
+            });
+        } catch (e) {
+            // now try system-wide one below
+        }
 
         this._setting = new Gio.Settings({
```

Comment:
shouldn't this be in the catch block? this way we are always overwriting what we set on line 29

Reply:
Good catch! Woops, forgot to include a `return;` statement in the `try { } catch {}` block.

Reply:
Updated the code.